src/ansible_navigator/utils/key_value_store.py

```diff
@@ -61,28 +87,33 @@ def iteritems(self):
         for row in cursor.execute("SELECT key, value FROM kv"):
             yield row[0], row[1]
 
-    def keys(self):
+    def keys(self) -> KVSKeysView:
         """Return all keys in the key-value store.
 
         :returns: All the keys
         """
-        return list(self.iterkeys())
+        return KVSKeysView(self)
 
-    def values(self):
+    def values(self) -> KVSValuesView:
         """Return all values in the key-value store.
 
         :returns: All the values
         """
-        return list(self.itervalues())
+        return KVSValuesView(self)
 
-    def items(self):
+    def items(self) -> KVSItemsView:
         """Return all items from the key-value store.
 
         :returns: The key-value store as items (key, value)
         """
-        return list(self.iteritems())
-
-    def __contains__(self, key):
+        return KVSItemsView(self)
+
+    # mypy complains about this 'str' because it's more specific than the 'object'
+    # that dict's Mapping superclass wants. However, it's correct in our case.
+    # Our sqlite table expects string keys (column type 'text'). It's an error to
+    # pass something else. If we pass something too weird, the sqlite3 lib will
+    # throw an error at runrtime.
```

Comment:
Should we check isinstance(key, str) here and raise our own exception?  Since we know we only want a str....

Reply:
Good point.... mypy does check the share directory so it should catch someone trying to stuff something odd in :)